ts/features/bonus/siciliaVola/components/SvVoucherListFilters.tsx

```diff
@@ -0,0 +1,241 @@
+import * as React from "react";
+import { useState } from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { Body, Container, Left, ListItem, Right, View } from "native-base";
+import {
+  FlatList,
+  Keyboard,
+  ListRenderItemInfo,
+  SafeAreaView,
+  ScrollView
+} from "react-native";
+import AppHeader from "../../../../components/ui/AppHeader";
+import ButtonDefaultOpacity from "../../../../components/ButtonDefaultOpacity";
+import IconFont from "../../../../components/ui/IconFont";
+import { H5 } from "../../../../components/core/typography/H5";
+import I18n from "../../../../i18n";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H2 } from "../../../../components/core/typography/H2";
+import ItemSeparatorComponent from "../../../../components/ItemSeparatorComponent";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../bonusVacanze/components/buttons/ButtonConfigurations";
+import { LabelledItem } from "../../../../components/LabelledItem";
+import DateTimePicker from "../../../../components/ui/DateTimePicker";
+import { GlobalState } from "../../../../store/reducers/types";
+import { possibleVoucherStateSelector } from "../store/reducers/voucherList/possibleVoucherState";
+import { isReady } from "../../bpd/model/RemoteValue";
+import { StatoVoucherBean } from "../../../../../definitions/api_sicilia_vola/StatoVoucherBean";
+import { H4 } from "../../../../components/core/typography/H4";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import { svResetFilter, svSetFilter } from "../store/actions/voucherList";
+import { FilterState } from "../store/reducers/voucherList/filters";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps> & {
+    onClose: () => void;
+    onConfirm: () => void;
+  };
+
+/**
+ * Component used to show and select the possible voucher state option using a radio button
+ * @param text
+ * @param value
+ * @param onPress
+ * @param checked
+ * @constructor
+ */
+const PossibleVoucherStateOption = ({
+  text,
+  value,
+  onPress,
+  checked
+}: {
+  text: string;
+  value: number;
+  onPress: (value: number) => void;
+  checked: boolean;
+}) => (
+  <ListItem
+    style={{
+      flexDirection: "row",
+      justifyContent: "space-between",
+      paddingVertical: 12
+    }}
+    onPress={() => onPress(value)}
+  >
+    <H4 weight={checked ? "SemiBold" : "Regular"} color={"bluegreyDark"}>
+      {text}
+    </H4>
+    <IconFont
+      name={checked ? "io-radio-on" : "io-radio-off"}
+      size={22}
+      color={checked ? IOColors.blue : IOColors.bluegrey}
+    />
+  </ListItem>
+);
+
+const SvVoucherListFilters: React.FunctionComponent<Props> = (props: Props) => {
```

Comment:
From the accessibility point of view this screen is missing some elements (eg: textinput should be like the credit card insert screen, the radio button are not recognized (selected / unselected) and the date picker seems not accessible on iOS). I don't know if you want to address now this issues or create a story to refine the screen in another iteration.

Reply:
I tried the date picker on IOS and it seems to recognize the slider.
Btw I've already create a story and we'll do the check of all the flow with the designer team.
Thank you